### DISCUSSION TITLE
Is this a bug? Weird issue when chanting the size of the window

### DISCUSSION THREAD

**Discussion by Ncage1974** [ghostty-org/ghostty#3851] [Q&A] [answered]
I'm using a apple silicon macbook pro & have weird issue when i change the size of the terminal window.  Here is a screenshot:

<img width="1076" alt="image" src="https://github.com/user-attachments/assets/90ef1d10-e3ac-49ea-bf3a-f23774eed6a9" />    

Everything looks normal until I change the size of the window & then i see all the error text in the screenshot. The application with all the exceptions works fine. Same thing doesn't happen in either kitty or alacritty.  The version i'm using is `1.0.0`. If i close ghostty & reopen and change the size of the window it will happen over & over. Configuration wise the only thing i've changed is the size of the font to 14. Is this a ghostty bug?

**Comment by 00-kat** [comment]
Huh another one of these.  Looks like it might be a bug in either `thefuck` or `colorama`...

[Someone else](https://discord.com/channels/1005603569187160125/1322532743829127239) had this issue with the tide prompt and conda recently, maybe the solution at https://github.com/IlanCosman/tide/issues/143#issuecomment-923107133 might work for you.

Edit: I found https://github.com/tartley/colorama/issues/302 but I don't know if that's relevant.

Edit 2: I'm not that familiar with `colorama`'s codebase, but I wonder why `ansitowin32.py` is even running if you're not on Windows since if I remember correctly they had a check there.  I also wonder if Ghostty is invalidating some pipe or the other.

**Reply by Ncage1974** [reply]
TY. Would you recommend just posting an issue on `thefuck` repo & going from there? See if the redirect me to colorama or somewhere else? I am using tide & fish shell.

**Reply by 00-kat** [reply, marked as answer]
Update: someone else in that thread is having this issue as well, but with something rust related which tries to write completions with `clap_complete` to... somewhere?

It might be a Tide or Ghostty bug instead, but I'm not sure.
